.github/policies/resourceManagement.yml

```
@@ -0,0 +1,43 @@
+id: 
+name: GitOps.PullRequestIssueManagement
+description: GitOps.PullRequestIssueManagement primitive
```

The reference that you used had this policy generated when we migrated from the previous automation. For a new policy, we can name it more appropriately and have as many `.yml` files in here as we want, with each of them being purpose-specific.

The following suggestion should be paired with renaming the file to `breaking-change.yml`.

```suggestion
name: breaking-change
description: When an issue or PR is labeled as a breaking change, add the label that a breaking change doc is needed, and comment with the breaking change process.
```